File: Submissions/LyftLevel5Challenge2018-FinalRound/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  long long n;
  cin >> n;

  string answer = "White";

  long long x, y;
  cin >> x >> y;

  long long w = x + y - 2;
  long long b = n * 2 - x - y;
  
  if (b < w) {
    answer = "Black";
  }

  cout << answer << endl;

}

File: Submissions/LyftLevel5Challenge2018-FinalRound/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    long n = sc.nextLong();
    long x = sc.nextLong();
    long y = sc.nextLong();
    sc.close();

    String answer = "White";
    
    long w = x + y - 2;
    long b = n * 2 - x - y;
  
    if (b < w) {
      answer = "Black";
    }
    
    System.out.println(answer);
  
  }

}